docs/ops/broadcast_rules.md

```diff
@@ -0,0 +1,9 @@
+# Broadcast rules for elementwise ops {#openvino_docs_ops_broadcast_rules}
+
+The purpose of this document is to provide set of common rules which are applicable for ops using broadcasting.
+
+## Rules
+
+1. * **Broadcast range of values**:
+    * *none* - no auto-broadcasting is allowed, all input shapes must match,
+    * *numpy* - numpy broadcasting rules, description is available in [Broadcast_1](movement/Broadcast_1.md).
```

Comment:
The document should have in detailed explanation of all possible broadcasting rules with example. Similar to the https://numpy.org/doc/stable/user/basics.broadcasting.html. 

So remove all details from the `movement/Broadcast_1.md` and move keep it in this dedicated document.

Reply:
@ntyukaev , @andrew-zaytsev, please, review this new document once it is done.